gapic/core/artman_logging_types.yaml

```diff
@@ -0,0 +1,16 @@
+common:
+  api_name: service-logging-types
```

Comment:
`logging-external-types` or `logging-extra-types` seems like a better name for this package. I stumble on the meaning of the `service` token when mentally parsing `grpc-google-service-logging-types-v1`. Also, it would be good if this is consistent with the service config filename.